contracts/src/periphery/EigenDACertVerifierV1.sol

```diff
@@ -0,0 +1,117 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.9;
+
+import {IEigenDACertVerifierV1} from "src/interfaces/IEigenDACertVerifierV1.sol";
+import {IEigenDAThresholdRegistry} from "../interfaces/IEigenDAThresholdRegistry.sol";
+import {IEigenDABatchMetadataStorage} from "../interfaces/IEigenDABatchMetadataStorage.sol";
+import {IEigenDASignatureVerifier} from "../interfaces/IEigenDASignatureVerifier.sol";
+import {EigenDACertVerificationV1Lib as CertV1Lib} from "src/libraries/EigenDACertVerificationV1Lib.sol";
+import {EigenDACertVerificationV2Lib as CertV2Lib} from "src/libraries/EigenDACertVerificationV2Lib.sol";
+import {OperatorStateRetriever} from "../../lib/eigenlayer-middleware/src/OperatorStateRetriever.sol";
+import {IRegistryCoordinator} from "../../lib/eigenlayer-middleware/src/RegistryCoordinator.sol";
+import {IEigenDARelayRegistry} from "../interfaces/IEigenDARelayRegistry.sol";
+import "../interfaces/IEigenDAStructs.sol";
+
+/**
+ * @title A CertVerifier is an immutable contract that is used by a consumer to verify EigenDA blob certificates
+ * @notice For V2 verification this contract is deployed with immutable security thresholds and required quorum numbers,
+ *         to change these values or verification behavior a new CertVerifier must be deployed
+ */
+contract EigenDACertVerifierV1 is IEigenDACertVerifierV1 {
+    IEigenDAThresholdRegistry public immutable eigenDAThresholdRegistryV1;
+
+    IEigenDABatchMetadataStorage public immutable eigenDABatchMetadataStorageV1;
+
+    /**
+     * @notice Constructor for the EigenDA V1 certificate verifier
+     * @param _eigenDAThresholdRegistryV1 The address of the EigenDAThresholdRegistry contract
+     * @param _eigenDABatchMetadataStorageV1 The address of the EigenDABatchMetadataStorage contract
+     */
+    constructor(
+        IEigenDAThresholdRegistry _eigenDAThresholdRegistryV1,
+        IEigenDABatchMetadataStorage _eigenDABatchMetadataStorageV1
+    ) {
+        eigenDAThresholdRegistryV1 = _eigenDAThresholdRegistryV1;
+        eigenDABatchMetadataStorageV1 = _eigenDABatchMetadataStorageV1;
+    }
+
+    /**
+     * @notice Verifies a the blob cert is valid for the required quorums
+     * @param blobHeader The blob header to verify
+     * @param blobVerificationProof The blob cert verification proof to verify
+     */
+    function verifyDACertV1(BlobHeader calldata blobHeader, BlobVerificationProof calldata blobVerificationProof)
```

Comment:
nit - why have additional redundancy with V1 suffix given versioning is already declared in contract name?

Reply:
This is to maintain compatibility with the old CertVerifier interface. Note that the CertVerifier inherits CertVerifierV1 and CertVerifierV2. In this way, the interface of the CertVerifier contract does not change, and so this PR is not a breaking change.

Reply:
Note that if the suffix was not here, aka it was just called verifyDACert, then this function selector would need to be exposed in inheriting contracts! which is a problem if we are trying to inherit and expose functions from both V1 and V2.